prow/jobs/kyma/kyma-compass-provisioner-tests.yaml

```diff
@@ -0,0 +1,102 @@
+---
+test_infra_ref: &test_infra_ref
+  org: kyma-project
+  repo: test-infra
+  path_alias: github.com/kyma-project/test-infra
+
+kyma_ref: &kyma_ref
+  org: kyma-project
+  repo: kyma
+  path_alias: github.com/kyma-project/kyma
+
+# Kubectl 1.14, CLUSTER_VERSION=1.14
+k8s_1-14: &k8s_1-14
+  image: eu.gcr.io/kyma-project/test-infra/kyma-cluster-infra:v20200124-8f253e51
+
+gke_integration_job_labels_template: &gke_integration_job_labels_template
+  preset-kyma-keyring: "true"
+  preset-kyma-encryption-key: "true"
+  preset-kms-gc-project-env: "true"
+  preset-kyma-guard-bot-github-token: "true"
+  preset-sa-gke-kyma-integration: "true"
+  preset-gc-compute-envs: "true"
+  preset-gc-project-env: "true"
+  preset-docker-push-repository-gke-integration: "true"
+  preset-dind-enabled: "true"
+  preset-kyma-artifacts-bucket: "true"
+  preset-gardener-azure-kyma-integration: "true"
+
+presubmits: # runs on PRs
+  kyma-project/kyma:
+  - name: pre-master-kyma-gke-compass-provisioner-tests
+    optional: true
+    decorate: true
+    path_alias: github.com/kyma-project/kyma
+    skip_report: false
+    max_concurrency: 10
+    spec:
+      containers:
+        - <<: *k8s_1-14
+          securityContext:
+            privileged: true
+          command:
+            - "bash"
+          args:
+            - "-c"
+            - "${KYMA_PROJECT_DIR}/test-infra/prow/scripts/cluster-integration/kyma-gke-compass-provisioner-tests.sh"
+          env:
+            - name: CLOUDSDK_COMPUTE_ZONE
+              value: "europe-west4-b"
+          resources:
+            requests:
+              memory: 200Mi
+              cpu: 80m
+    branches:
+      - ^master$
+    # following regexp won't start build if only Markdown files were changed
+    run_if_changed: "resources/compass/charts/provisioner"
```

Comment:
The job will not run when we, for example, bump Provisioner image, as the image version is stored in `resources/compass/values.yaml`. Maybe it is ok to prevent too frequent runs but then we need to remember to trigger the job manually.

Reply:
Changed to "^(?:(resources\/compass\/charts\/provisioner)|(resources\/compass\/values.yaml))", so it triggers on any change in provisioner chart or values.yaml... At least in theory 🤣

Reply:
Left as was, because it will remain optional, and would be triggered only for provisioner changes